lib/pure/collections/sequtils.nim

```diff
@@ -326,7 +326,6 @@ func distribute*[T](s: seq[T], num: Positive, spread = true): seq[seq[T]] =
   if num < 2:
     result = @[s]
     return
-  let num = int(num) # XXX probably only needed because of .. bug
```

Comment:
This was introduced by this PR https://github.com/nim-lang/Nim/commit/51034d7e3ac592d111e62b023c55e001bd3d2f94 on 8 Mar 2015.